test/util/confirmRovingTabindex.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const { By } = require('selenium-webdriver');
+
+/**
+ * Confirm the roving tabindex has been initialized for a list of elements.
+ * Contains as many assertions as the number of elements returned by element selector squared.
```

Comment:
Oh math, fun! Would it be useful to include a utility to calculate this?

Reply:
haha I ultimately realized that math is not a good idea -- and removed in this PR: #787

Previously the util functions had multiple ava asserts that had to be counted for the "t.plan" -- now all util functions will only have one ava assert.